core/connection.go

```diff
@@ -50,11 +64,16 @@ func newConnection(
 	}
 }
 
-// ID returns the connection ID.
-func (c *Connection) ID() string {
+// ID returns the increment ID.
```

Comment:
```suggestion
// ID returns the connection ID.
```

this `ID` property always represents the connection ID, developers do not need to understand it's auto-incremented and focus on its use.